### ISSUE TITLE
Empty ref object throw an error

### ISSUE THREAD

**Issue by baptistecs** [preactjs/preact#3499]
- [x] Check if updating to the latest Preact version resolves the issue

**Describe the bug**

Regarding this fix merged yesterday: https://github.com/preactjs/preact/pull/3494

If we do this:
```jsx
const MyComponent = forwardRef(({ myProp = true }, { myRefA, myRefB }) => { ... });
<MyComponent ref={{}} />
```
It will generate this error:
`Uncaught (in promise) TypeError: Cannot destructure property 'myRefA' of 'object null' as it is null.`

While it's working fine with React

**Comment by marvinhagemeister** [comment]
This is resolved by #3497 . We'll cut a patch release soon.

**Comment by baptistecs** [comment]
❤️ Thank you!

**Comment by JoviDeCroock** [comment]
Fixed in 10.7.1
